dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -744,6 +744,7 @@ class BGPPeerMgrBase(Manager):
         :param data: the data associated with the change
         """
         vrf, nbr = self.split_key(key)
+        key = (vrf, nbr)
```

Comment:
Seems odd to redefine key here. Maybe just change the following line to `if (vrf, nbr) not in self.peers:`. Will this suffice?

Reply:
Agree. It is better to use different name. 
key in the param list is a string
key which is defined here is a tuple of two strings.

Reply:
@pavel-shirshov @JLevesque84 updated to use "peer_key"

Reply:
@abdosi; FYI, you tagged somebody that isn't me, but has a similar username :)